src/Settings/RTK.SettingsGroup.json

```diff
@@ -3,9 +3,17 @@
     "fileType":     "FactMetaData",
     "QGC.MetaData.Facts":
 [
+{
+    "name":                 "baseReceiverManufacturers",
+    "shortDesc":            "GPS manufacturers for settings",
+    "type":                 "uint8",
+    "enumStrings":          "Standard,All,Trimble,Septentrio,Femtomes,UBlox",
```

Comment:
What is the meaning of `Standard` (I see what it means in the code, but from a user perspective)?
And shouldn't this be determined automatically based on the connected gps type?

Reply:
Are these values "1,31,3,5,9,17" somewhere in a driver code enum somewhere? If so, can that be used on the C++ side instead of all the magic numbers in the code? If not, can an enum be added to RTKSettings such that we don't use magic numbers on the c++ side?